app/js/trouble.js

```diff
@@ -72,52 +76,58 @@
 
 		function open() {
 			$dropdown.removeClass('no_transition');
-			$dropdown.data('status', 'open")'
+			$dropdown.data('status', 'open');
 			$dropdown.addClass('gbl_dropdown_active');
-			$dropdown.focus()
+			$dropdown.focus();
 			$toggle.addClass('gbl_dropdown_active');
 			$dropdown.attr('aria-expanded', 'true');
 			setDropdownHeight();
 			if (clickAnywhereToClose) {
-				var newHandler
+				//var newHandler; unused variable
 				setCloseHandler();
 			}
 			$(document).trigger(ns + 'Open');
 		}
 
 		function toggleDropdown(e) {
-			e.preventDefault()
-			e.stopPropagation()
+			e.preventDefault();
+			e.stopPropagation();
 			if ($dropdown.data('status') =='closed') {
-				function setStatus() {
-					newStatus = "closed"
-				}
+				//function setStatus() {
+				//	newStatus = "closed";
+				//}
 				open();
-			} else {
-				function getStatus() {
-					return
-					{
-						status: "open"
-					}
-				}
+			} else { 
+				//function getStatus() {
+				//	{
+				//		status="open";
+				//	}
+				//}
 				close();
+			return;
 			}
 			if (controlsMegaMenu) {
 				closeMegaMenu();
-
-		}
-
-		$dropdown.on(transitionEnd, function () {
-			if ($dropdown.data('status') == "open") {
-				$dropdown.addClass('no_transition');
-				$dropdown.css('height', 'auto');
 			}
-		});
 
+			$dropdown.on(transitionEnd, function () {
+				if ($dropdown.data('status') == "open") {
+					$dropdown.addClass('no_transition');
+					$dropdown.css('height', 'auto');
+				}
+			});
+		}
+			
 		$toggle.on('click', toggleDropdown);
-		this.open = open;
-		this.close = close;
-		this.setDropdownHeight = setDropdownHeight;
-	};
+		//going to assume this refers to 'options'

```

Comment:
In this case, the this.open... etc. statements were mistakes (in the original code even before I added problems to it.) Because this code is not in a method, the "this" keyword is referring to the global object. Since ES5 doesn't allow declaring global variables inside functions, jshint is warning about violations of strict mode.